### PR TITLE
Automatic protobuf files generation

### LINKED ISSUE #317: Consider avoiding store generated protobuf files in git.
Consider opportunity to generate protobuf files (that are generated from protocols) during installation process, instead keeping them in git.

### PR DESCRIPTION
**Pull request by itrushkin**
- Removes *.pb2 files from `openfl/protocols` folder.
- Adds automatic protobuf compilation on `pip install` command.

Closes #317.

### CODE REVIEW

**Comment by itrushkin** [comment]
Jenkins please retry a build

**Comment by itrushkin** [comment]
Jenkins please retry a build

File: setup.py [hunk 1]
```diff
@@ -2,8 +2,42 @@
...
 from setuptools import setup
+from setuptools.command.build_py import build_py
+
+
+class GRPCBuildPyCommand(build_py):
+    """Command to generate project *_pb2.py modules from proto files."""
+
+    def run(self):
+        """Build Python and GRPC modules."""
+        from subprocess import check_call
```
Comment (aleksandr-mokrov) [inline setup.py]: What is the reason to import inside? Probably it should be moved to the top?

**Review by aleksandr-mokrov — APPROVED** [review]

File: setup.py (hunk 1 above)
Comment (itrushkin) [reply setup.py]: Correct. I moved these import lines to the top.

**Comment by itrushkin** [comment]
Jenkins please retry a build

**Review by dmitryagapov — APPROVED** [review]

File: setup.py [hunk 2]
```diff
@@ -3,7 +3,42 @@
...
 from setuptools import setup
+from setuptools.command.build_py import build_py
+
+
+class GRPCBuildPyCommand(build_py):
+    """Command to generate project *_pb2.py modules from proto files."""
+
+    def run(self):
+        """Build Python and GRPC modules."""
+        check_call([executable, '-m', 'pip', 'install', 'grpcio-tools~=1.34.0'])
```
Comment (igor-davidyuk) [inline setup.py]: are protobuf files built before we install requirements? 
In other case I do not see why we call `pip` here explicitly

Comment (itrushkin) [reply setup.py]: Yes, on this step we do not have any of the requirements installed.

**Comment by itrushkin** [comment]
Jenkins please retry a build

**Comment by itrushkin** [comment]
retest this please

**Comment by itrushkin** [comment]
Jenkins please retry a build

**Review by igor-davidyuk — APPROVED** [review]

**Pull request merged**